From ed204e17e230f2018fac626b920e5181710a22f3 Mon Sep 17 00:00:00 2001
From: tetsuji tsuda <tetsuji.tsuda.xd@renesas.com>
Date: Wed, 20 Jul 2016 12:15:58 +0900
Subject: [PATCH 1625/2066] arm64: dts: r8a7796: Add QoS Driver node at Device
 Tree

This patch adds QoS driver support for R8A7796 SoC.

Signed-off-by: Tetsuji Tsuda <tetsuji.tsuda.xd@renesas.com>
Signed-off-by: Takeshi Kihara <takeshi.kihara.df@renesas.com>
[zou:Original patch taken from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git
 v4.9/rcar-3.5.1]
Signed-off-by: zou cao <cao.zou@windriver.com>
---
 arch/arm64/boot/dts/renesas/r8a7796.dtsi |    4 ++++
 1 files changed, 4 insertions(+), 0 deletions(-)

diff --git a/arch/arm64/boot/dts/renesas/r8a7796.dtsi b/arch/arm64/boot/dts/renesas/r8a7796.dtsi
index b8da7da..61297e1 100644
--- a/arch/arm64/boot/dts/renesas/r8a7796.dtsi
+++ b/arch/arm64/boot/dts/renesas/r8a7796.dtsi
@@ -2388,6 +2388,10 @@
 			power-domains = <&sysc R8A7796_PD_3DG_B>;
 		};
 
+		qos@e67e0000 {
+			compatible = "renesas,qos";
+		};
+
 		vsp@fe960000 {
 			compatible = "renesas,vspm-vsp2";
 			reg = <0 0xfe960000 0 0x8000>;
-- 
1.7.5.4

